### PR TITLE
SP-8108: Shared evt db

### PR DESCRIPTION
**Pull request by rafalgajda-silvair**
This patch adds functionality needed for concurrent service operation with independent VEN clients accessing the same Event DB.

### CODE REVIEW

File: oadr2/event.py [hunk 1]
```diff
@@ -456,7 +457,7 @@ def remove_events(self, evt_id_list):
         '''
         Remove a list of events from our internal member dictionary
 
-        event_id_list - List of Event IDs 
+        event_id_list - List of Event IDs
```
Comment (michallowasrzechonek-silvair) [inline oadr2/event.py:460]: Please fix formatting in a separate commit.

File: oadr2/poll.py [hunk 2]
```diff
@@ -75,23 +77,42 @@ def __init__(self, event_config, vtn_base_uri,
         self.__password = password
 
         self.poll_thread = None
-        start_thread = bool(start_thread)
-        self._init_client(start_thread)
+        if start_thread:  # this is left for backward compatibility
+            self.start()
 
-        logging.info(" +++++++++++++++ OADR2 module started ++++++++++++++ ")
+        logging.info(" +++++++++++++++ OADR2 module started ++++++++++++++ !!")
```
Comment (michallowasrzechonek-silvair) [inline oadr2/poll.py]: !!

File: oadr2/poll.py [hunk 3]
```diff
@@ -75,23 +77,42 @@ def __init__(self, event_config, vtn_base_uri,
...
+        self.poll_thread.start()
+        logging.info("Polling thread started")
+
+    def stop(self):
+        '''
+        Stops polling without stopping event controller
+        :return:
+        '''
+        if self.poll_thread is not None:
+            self.poll_thread.join(2)  # they are daemons.
```
Comment (michallowasrzechonek-silvair) [inline oadr2/poll.py:109]: What should happen if this times out? Also, pass this as `timeout=2` for readability.

File: oadr2/poll.py [hunk 4]
```diff
@@ -121,7 +142,12 @@ def poll_vtn_loop(self):
             except Exception as ex:
                 logging.exception("Error in OADR2 poll thread: %s", ex)
 
-            self._exit.wait(self.vtn_poll_interval)
+            self._exit.wait(
```
Comment (michallowasrzechonek-silvair) [inline oadr2/poll.py]: This belongs to a separate pull request.

File: oadr2/event.py (hunk 1 above)
Comment (rafalgajda-silvair) [reply oadr2/event.py:460]: Fixed

File: oadr2/poll.py (hunk 2 above)
Comment (rafalgajda-silvair) [reply oadr2/poll.py]: Fixed

File: oadr2/poll.py (hunk 3 above)
Comment (rafalgajda-silvair) [reply oadr2/poll.py:109]: This is a method used by original authors to stop threads. It was just split here to only stop polling or control threads

File: oadr2/poll.py (hunk 4 above)
Comment (rafalgajda-silvair) [reply oadr2/poll.py]: Removed

File: oadr2/poll.py (hunk 3 above)
Comment (michallowasrzechonek-silvair) [reply oadr2/poll.py:109]: :ok_hand:
